examples/visualization/plot_evoked_arrowmap.py

```diff
@@ -0,0 +1,34 @@
+"""
+=============================================
+Plotting topographic arrowmaps of evoked data
+=============================================
+
+Load evoked data and plot arrowmaps along with the topomap for selected time
+points. Arrowmap is based upon Hosaka-Cohen transformation and represents
+actual current underneath the MEG sensors, they are poor man MNE
+Cohen, D.; Hosaka, H. (1976). "Part II: Magnetic field produced by a current
+dipole". Journal of Electrocardiology
```

Comment:
see how we format ref as in for example:

https://github.com/mne-tools/mne-python/blob/master/examples/inverse/plot_gamma_map_inverse.py

Reply:
Thanks, fixed.